concourse/scripts/pipelines-bosh-cloudfoundry.sh

```diff
@@ -54,6 +55,14 @@ prepare_environment() {
 
   download_git_id_rsa
   get_git_concourse_pool_clone_full_url_ssh
+
+  if [ "${ENABLE_DATADOG:-}" = "true" ]; then
+    export PASSWORD_STORE_DIR=~/.paas-pass
```

Comment:
this is only now for current datadog dev/test account. Once we have a final account we might have a different set(s) of credentials. We'll have to update the credential getting part then.

regarding env. vars you are expected to set when deploying to dev - I think env. name is enough. You might want to add BRANCH if you want to deploy from a different branch.